eng/Versions.props

```diff
@@ -6,8 +6,6 @@
     <MajorVersion>5</MajorVersion>
     <MinorVersion>0</MinorVersion>
     <PatchVersion>0</PatchVersion>
-    <!-- Always use shipping version instead of dummy version -->
-    <DotNetUseShippingVersions>true</DotNetUseShippingVersions>
```

Comment:
Can we leave this set to true when we're in an Official build? So that package versions are unique?

Reply:
Not necessary. https://github.com/dotnet/arcade/blob/bca68c2df2c6230389f10def3880a61db266c6c1/src/Microsoft.DotNet.Arcade.Sdk/tools/Version.BeforeCommonTargets.targets#L39

Reply:
Awesome, arcade is ahead of me 😄